addons/account/models/account_move.py

```diff
@@ -3702,17 +3702,28 @@ def _notify_get_groups(self, msg_vals=None):
         in account. Customer and portal group have probably no right to see
         the document so they don't have the access button. """
         groups = super(AccountMove, self)._notify_get_groups(msg_vals=msg_vals)
+        recipient_group = None
 
         self.ensure_one()
         if self.move_type != 'entry':
+            access_link = None
             for group_name, _group_method, group_data in groups:
-                if group_name in ('portal_customer', 'customer'):
+                if group_name == 'portal_customer':
                     group_data['has_button_access'] = True
+                    access_link = group_data['button_access']['url']
+
+            button_access = {'url': access_link} if access_link else {}
```

Comment:
Would be nice to add a general high level comment explaining why we have to do this